python/tests/experimental/test_user.py

```diff
@@ -0,0 +1,47 @@
+from unittest.mock import Mock, patch
+
+import pytest
+
+from numerous.collection import NumerousCollection
+from numerous.user import User
+
+
+@pytest.fixture
+def mock_framework_detector() -> Mock:
+    return Mock()
+
+@pytest.fixture
+def user(mock_framework_detector: Mock) -> User:
+    user = User()
+    user.framework_detection = mock_framework_detector
+    return user
+
+def test_user_id_with_cookie(user: User, mock_framework_detector: Mock) -> None:
+    mock_framework_detector.get_cookies.return_value =\
+          {"numerous_user_id": "test-user-123"}
```

Comment:
A controversial line break :smile_cat: Normally we'd rather just "open" the dict, e.g.

```python
some_thing = {
   "key": "value",
}
```

Reply:
Thanks, will remember for next time this occurs